### ISSUE TITLE
Add accessibility label for video blocks

### ISSUE THREAD

**Issue by barklund** [ampproject/amp-wp#3186]
_Spawned from #2983_

As a user I want to be able to see and edit the accessibility label for video elements directly in the block editor for a given video placed in a story.

The value should be pre-populated with the value from the `title` attribute from the media editor.

The field should be a simple single-line text field placed in the _Video Settings_ section of the block element editor with a descriptive label.

The value should be saved with the individual page with the given media element and not on the media element itself. If multiple pages/stories use the same media element, they can have different accessibility labels.

The field should be rendered as the `aria-label` attribute on the `<amp-video>` node  as part of the AMP story output.

**Comment by csossi** [comment]
Verified in QA

**Comment by barklund** [comment]
Screencast: https://youtu.be/cp8u1IqPiEQ

**Comment by pbakaus** [comment]
This is pretty cool!
